OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -657,7 +657,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                                 <item xml:id="a4">
                                     <locus target="#5r"/>
-                                    <desc type="Condemnation">Note mentioning the name of the manuscript's buyer, <persName ref="PRS13148AmdaM"/>,  and condemning those who stole it.</desc>
+                                    <desc type="Condemnation">Note mentioning the name of the manuscript’s buyer, <persName ref="PRS13148AmdaM"/>, and condemning those who stole it.</desc>
```

Comment:
(not your change, but I think the note condemns "whoever might steal" the manuscript – I take it as a kind of irrealis perfect, similar to what is used in conditional clauses)